scripts/build/Platform/Windows/installer_windows.cmd

```diff
@@ -17,14 +17,6 @@ IF NOT EXIST %OUTPUT_DIRECTORY% (
 )
 PUSHD %OUTPUT_DIRECTORY%
 
-REM Override the temporary directory used by wix to the workspace (if we have a WORKSPACE_TMP)
-IF NOT "%WORKSPACE_TMP%"=="" (
-    SET "WIX_TEMP=!WORKSPACE_TMP!/wix"
-    IF NOT EXIST "!WIX_TEMP!" (
-        MKDIR "!WIX_TEMP!"
-    )
-)
```

Comment:
If you haven't already, might want to run the installer job on your branch just to confirm this change doesn't mess with Jenkins.

Reply:
Yes, run a branch build of the installer and validated that it works